src/test/java/io/github/vlsergey/springdatarestutils/EntityToSchemaMapperTest.java

```diff
@@ -65,15 +68,15 @@ void testLink() throws Exception {
 		"  type:\n" + //
 		"    type: string\n" + //
 		"    nullable: false\n" + //
-		"", json);
+		""), yamlMap);
     }
 
     @Test
     void testMapAsDataItem() throws Exception {
 	final Schema<?> schema = mapper.mapEntity(TestEntity.class, ClassMappingMode.DATA_ITEM, RequestType.RESPONSE);
-	String json = SchemaUtils.writeValueAsString(false, schema);
+	Map<String, Object> json = yaml.load(SchemaUtils.writeValueAsString(false, schema));
```

Comment:
If you change above to be `yamlMap`, you want to change all occurrence to keep consistent.